tests/bot/exts/info/test_information.py

```diff
@@ -422,7 +424,7 @@ async def test_create_user_embed_uses_blurple_colour_when_user_has_no_roles(self
         """The embed should be created with a blurple colour if the user has no assigned roles."""
         ctx = helpers.MockContext()
 
-        user = helpers.MockMember(id=217)
+        user = helpers.MockMember(id=217, colour=discord.Colour.blurple())
```

Comment:
Set the colour to `0` here, since it is testing for the embed colour if the user has no assigned roles, so they will only have the `@everyone` role.
```suggestion
        user = helpers.MockMember(id=217, colour=0)
```

Reply:
Resolved all comments in 835a0a6f4a45018d21dacbfbf69afe07361155aa!